php/utils-wp.php

```diff
@@ -220,14 +220,14 @@ function wp_get_cache_type() {
 
 	if ( ! empty( $_wp_using_ext_object_cache ) ) {
 		// Test for Memcached PECL extension memcached object cache (https://github.com/tollmanz/wordpress-memcached-backend)
-		if ( isset( $wp_object_cache->m ) && is_a( $wp_object_cache->m, 'Memcached' ) ) {
+		if ( isset( $wp_object_cache->m ) && ( $wp_object_cache->m instanceof \Memcached ) ) {
 			$message = 'Memcached';
 
 			// Test for Memcache PECL extension memcached object cache (http://wordpress.org/extend/plugins/memcached/)
 		} elseif ( isset( $wp_object_cache->mc ) ) {
 			$is_memcache = true;
 			foreach ( $wp_object_cache->mc as $bucket ) {
-				if ( ! is_a( $bucket, 'Memcache' ) && ! is_a( $bucket, 'Memcached' ) ) {
+				if ( ( ! $bucket instanceof \Memcache ) && ( ! $bucket instanceof \Memcached ) ) {
```

Comment:
Parentheses are not needed here:
```suggestion
				if ( ! $bucket instanceof \Memcache && ! $bucket instanceof \Memcached ) {
```

Reply:
I'll change that of course, but here I think the expression is more readable with the parentheses.